tictactoe/tictactoe.js

```diff
@@ -0,0 +1,94 @@
+var origBoard;
+var player;
+const player1   = "X";
+const player2   = "O";
+const winCombos = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6],
+]
+const cells = document.querySelectorAll(".cell");
+startGame();
+
+function startGame() {
+    document.querySelector("#endgame").style.display = "none";
+    document.querySelector("#restart").style.display = "none";
+    player = player1 ;
+    document.getElementById("player").innerHTML = "X" ;
+    origBoard = Array.from(Array(9).keys());
+    for (var i = 0; i < cells.length; i++) {
+        cells[i].innerText = "";
+        cells[i].style.removeProperty("background-color");
```

Comment:
What does this line do?

Reply:
These lines take all of the tic tac toe squares and puts them back to having nothing in them.